css/style.css

```diff
@@ -0,0 +1,360 @@
+/* Style.css */
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+body {
+  font-family: helvetica;
+  font-size: 16px;
+}
+
+.stroke {
+  border: 3px solid #fff;
+  background: rgba(0, 0, 0, 0.75);
+  padding: 15px;
+  box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.5);
+}
+
+h1 {
+  font-size:50px;
```

Comment:
Now that we've gone through responsive fonts - consider using `rem`s or `em`s so you can easily scale your fonts on different devices.